Modules/Segmentation/Watersheds/include/itkWatershedSegmenter.hxx

```diff
@@ -176,9 +176,17 @@ Segmenter<TInputImage>::GenerateData()
   Self::MinMax(input, regionToProcess, minimum, maximum);
   // cap the maximum in the image so that we can always define a pixel
   // value that is one greater than the maximum value in the image.
-  if (NumericTraits<InputPixelType>::IsInteger CLANG_PRAGMA_PUSH CLANG_SUPPRESS_Wfloat_equal &&
+  if (NumericTraits<InputPixelType>::IsInteger
+// clang-format on
+CLANG_PRAGMA_PUSH
+CLANG_SUPPRESS_Wfloat_equal
+// clang-format off
+      &&
       maximum == NumericTraits<InputPixelType>::max())
-    CLANG_PRAGMA_POP { maximum -= NumericTraits<InputPixelType>::OneValue(); }
+// clang-format on
+CLANG_PRAGMA_POP
+// clang-format off
```

Comment:
Not at risk of nesting?

Reply:
Perhaps not, but macros that are required to be on their own line when defined, or irrelevant when empty,  should be protected with clang-format guards.